components/SubmissionForm.vue

```diff
@@ -198,7 +220,48 @@ const validateFields = (e: Event) => {
         e.preventDefault()
         return
     }
-    submissionStore.submit()
+    submitNewSubmission()
+}
+
+async function submitNewSubmission() {
+    const spokenLanguages: Locale[] = []
+
+    if (selectLanguage1.value !== '') {
+        spokenLanguages.push(selectLanguage1.value as Locale)
+    }
+
+    if (selectLanguage2.value !== '') {
+        spokenLanguages.push(selectLanguage2.value as Locale)
+    }
+
+    const newSubmission: MutationCreateSubmissionArgs
+    = { input: {
+        googleMapsUrl: location.value,
+        healthcareProfessionalName: `${firstName.value} ${lastName.value}`,
+        spokenLanguages: spokenLanguages,
+        notes: otherNotes.value
+    } }
+
+    const response = await submissionStore.createNewSubmission(newSubmission)
+    if (response?.errors?.length) {
+        response.errors.map(error => {
+            toast.error((t(`serverErrorMessages.${error.extensions.code}`)))
```

Comment:
should check if the code exists first. The server could have a new one that we haven't supported yet on the front end. In the case where it doesn't exist, we can just send a generic error message to the user and log the error code that was unrecognized.